src/GSF.CarbonAware/src/Json/DateTimeOffsetConverter .cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace GSF.CarbonAware.Json
+{
+    public class DateTimeOffsetConverter : JsonConverter<DateTimeOffset>
```

Comment:
@in4margaret, One more question: Is this a Mock/Integration test issue, only? Or do you see the same behavior using the datasources directly?

Reply:
We can see the same issue when data source, for example, JSON has timezones in the data.

Reply:
Perhaps this can be taken offline, but seems like this is not a json specific format issue. My concern is this logic is too specific to a data format rather than the raw value. Did you try to move this logic to the Model classes that contains the datetimeoffset instances? Rather than manipulating the json only? What would happen if the output is not json? That is what makes me think about this approach.

Reply:
Currently WebAPI output only in JSON. our code without this PR  depends on timezone that passed to our code from data source.

Reply:
@juzuluag changed to force conversion in DTO getters.